src/gui/src/tclCmdHighlighter.h

```diff
@@ -110,7 +110,12 @@ class TclCmdHighlighter : public QSyntaxHighlighter
     CommandRulePtr buildKeywordRule(const int command_id,
                                     const std::string& command,
                                     const std::string& start_of_command,
-                                    const std::string& end_of_command);
+                                    const std::string& end_of_command,
+                                    bool escape_cmd = true);
+    void buildKeywordsRule(std::vector<CommandRulePtr>& vec,
```

Comment:
can methods like this be const or even static (they don't seem to use class members at all).

Reply:
I don't see why not, I'll give it a go.